models/cow_protocol/ethereum/cow_protocol_ethereum_airdrop_claims.sql

```diff
@@ -2,45 +2,48 @@
     config(
         schema = 'cow_protocol_ethereum',
         alias='airdrop_claims',
+        tags=['dunesql'],
         materialized = 'table',
         file_format = 'delta',
         tags=['static'],
         unique_key = ['recipient', 'tx_hash', 'evt_index'],
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                 "project",
                                 "cow_protocol",
-                                \'["hildobby"]\') }}'
+                                \'["hildobby", "bh2smith"]\') }}'
     )
 }}
 
-{% set cow_token_address = '0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab' %}
```

Comment:
Wasn't sure if this was still a thing in dune sql and didn't bother risking it. Can put it back if necessary.

Reply:
this is still a thing. i believe the setup is:
`{% set cow_token_address = '0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab' %}`
`AND contract_address={{cow_token_address}}`

can you try this? maybe a quick compile and see if print the address without single quotes? then push commit back here?

Reply:
join condition too:
`AND pu.contract_address={{cow_token_address}}`